doc/manipulating_images/masker_objects.rst

```diff
@@ -504,3 +512,27 @@ seed position is used.
 .. topic:: **Examples**
 
   * :ref:`sphx_glr_auto_examples_03_connectivity_plot_sphere_based_connectome.py`
+
+
+Extraction of signals from surface images\  :class:`SurfaceMasker`, :class:`SurfaceLabelsMasker`, :class:`SurfaceMapsMasker`
+============================================================================================================================
+
+The purpose of :class:`SurfaceMasker`, :class:`SurfaceLabelsMasker`, :class:`SurfaceMapsMasker`
+is to mirror the capabilities of
+:class:`NiftiMasker`, :class:`NiftiLabelsMasker` and :class:`NiftiMapsMasker`
+but to extract data from :class:`~nilearn.surface.SurfaceImage`.
+
+They can perform data extraction from 1D surface data (n_vertices),
+2D surface data (n_vertices x samples)
```

Comment:
```suggestion
2D surface data (n_vertices, samples)
```

Reply:
oops sorry

I need to do another pass on this so I can integrate those suggestions